ruby/vampires.rb

```diff
@@ -0,0 +1,108 @@
+loop_run = 0
+	puts "How many employees will be processed?"
+	employees = gets.chomp.to_i
+	until loop_run == employees
+		loop_run+= 1
+	puts "What is your name?"
+	name = gets.chomp
+	flagged_names = "Tu Fang" || "Drake Cula"
+
+	puts "How old are you in years?"
+	age = gets.chomp.to_i
+
+		if age > 100
+			puts "That's suspicious"
+			age_input = true
+		else 
+			puts "Looking good!"
+			age_input = false
+		end
+
+	puts "What year were you born?"
+	year = gets.chomp.to_i
+	dob_time = 2017-year

```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.